src/meta/base_values.yaml

```diff
@@ -0,0 +1,514 @@
+minecraft:dead_tube_coral_fan: 1
+minecraft:dead_brain_coral_fan: 1
+minecraft:dead_bubble_coral_fan: 1
+minecraft:dead_fire_coral_fan: 1
+minecraft:dead_horn_coral_fan: 1
+
+# Transportation
+minecraft:saddle: 600
+minecraft:elytra: 20000
+minecraft:iron_horse_armor: 400
+minecraft:golden_horse_armor: 800
+minecraft:diamond_horse_armor: 1200
+
+# Netherite tools & armor (disabled because made of diamonds)
+minecraft:netherite_sword: -1
+minecraft:netherite_shovel: -1
+minecraft:netherite_pickaxe: -1
+minecraft:netherite_axe: -1
+minecraft:netherite_hoe: -1
+minecraft:netherite_helmet: -1
+minecraft:netherite_chestplate: -1
+minecraft:netherite_leggings: -1
+minecraft:netherite_boots: -1
+
+# Chaimail armor
+minecraft:chainmail_helmet: 500
+minecraft:chainmail_chestplate: 800
+minecraft:chainmail_leggings: 700
+minecraft:chainmail_boots: 400
+
+# Food
+minecraft:apple: 20
+minecraft:wheat_seeds: 2
+minecraft:porkchop: 20
+minecraft:cooked_porkchop: 20
+minecraft:enchanted_golden_apple: 1000
+minecraft:cod: 20
+minecraft:salmon: 20
+minecraft:tropical_fish: 20
+minecraft:pufferfish: 20
+minecraft:cooked_cod: 20
+minecraft:cooked_salmon: 20
+minecraft:melon_slice: 5
+minecraft:beef: 20
+minecraft:cooked_beef: 20
+minecraft:chicken: 20
+minecraft:cooked_chicken: 20
+minecraft:carrot: 10
+minecraft:potato: 10
+minecraft:baked_potato: 10
+minecraft:poisonous_potato: 1
+minecraft:rabbit: 20
+minecraft:cooked_rabbit: 20
+minecraft:mutton: 20
+minecraft:cooked_mutton: 20
+minecraft:chorus_fruit: 20
+minecraft:popped_chorus_fruit: 20
+minecraft:beetroot: 20
+minecraft:beetroot_seeds: 2
+minecraft:suspicious_stew: 10
+minecraft:sweet_berries: 5
+minecraft:glow_berries: 20
+
+# Loot
+minecraft:string: 10
+minecraft:feather: 5
+minecraft:gunpowder: 10
+minecraft:sponge: 200
+minecraft:wet_sponge: 200
+minecraft:egg: 5
+minecraft:ink_sac: 5
+minecraft:glow_ink_sac: 100
+minecraft:bone: 5
+minecraft:rotten_flesh: 5
+minecraft:ender_pearl: 80
+minecraft:blaze_rod: 30
+minecraft:ghast_tear: 80
+minecraft:spider_eye: 5
+minecraft:skeleton_skull: 40
+minecraft:wither_skeleton_skull: 300
+minecraft:zombie_head: 40
+minecraft:creeper_head: 40
+minecraft:dragon_head: 200
+minecraft:nether_star: 1000
+minecraft:rabbit_foot: 100
+minecraft:rabbit_hide: 5
+minecraft:totem_of_undying: 2000
+minecraft:shulker_shell: 250
+minecraft:trident: 1000
+minecraft:phantom_membrane: 100
+minecraft:nautilus_shell: 30
+minecraft:heart_of_the_sea: 1000
+
+# Materials
+minecraft:charcoal: 20
+minecraft:scute: 100
+minecraft:flint: 5
+minecraft:quartz: 20
+minecraft:amethyst_shard: 50
+minecraft:netherite_scrap: 1000
+minecraft:snowball: 1
+minecraft:clay_ball: 5
+minecraft:brick: 5
+minecraft:glowstone_dust: 10
+minecraft:nether_brick: 1
+minecraft:prismarine_shard: 20
+minecraft:prismarine_crystals: 20
+minecraft:firework_star: 10
+minecraft:dragon_breath: 100
+
+# Filled buckets
+minecraft:water_bucket: 300
+minecraft:lava_bucket: 300
+minecraft:milk_bucket: 300
```

Comment:
We can nerf these down, getting infinite lava and water is easy

Reply:
Right, they use the old pricing for base items, a few of those will have to be changed

Reply:
yep noticed